pkg/operator/starter.go

```diff
@@ -28,12 +28,29 @@ import (
 	operatorinformer "github.com/openshift/client-go/operator/informers/externalversions"
 	routeclient "github.com/openshift/client-go/route/clientset/versioned"
 	routeinformer "github.com/openshift/client-go/route/informers/externalversions"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/configobservation/configobservercontroller"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/deployment"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/endpointaccessible"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/ingressnodesavailable"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/ingressstate"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/metadata"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/payload"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/proxyconfig"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/readiness"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/routercerts"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/serviceca"
+	"github.com/openshift/cluster-authentication-operator/pkg/controllers/targetversion"
+	"github.com/openshift/cluster-authentication-operator/pkg/operator/assets"
+	oauthapiconfigobservercontroller "github.com/openshift/cluster-authentication-operator/pkg/operator/configobservation"
+	"github.com/openshift/cluster-authentication-operator/pkg/operator/revisionclient"
+	"github.com/openshift/cluster-authentication-operator/pkg/operator/workload"
 	"github.com/openshift/library-go/pkg/authentication/bootstrapauthenticator"
 	"github.com/openshift/library-go/pkg/controller/controllercmd"
 	libgoassets "github.com/openshift/library-go/pkg/operator/apiserver/audit"
 	workloadcontroller "github.com/openshift/library-go/pkg/operator/apiserver/controller/workload"
 	apiservercontrollerset "github.com/openshift/library-go/pkg/operator/apiserver/controllerset"
 	libgoetcd "github.com/openshift/library-go/pkg/operator/configobserver/etcd"
+	"github.com/openshift/library-go/pkg/operator/encryption"
```

Comment:
this is new, the rest was moved up.

Reply:
nit: please keep the local deps in a separate block

Reply:
it will be hard to keep track since the local deps match "github.com/opeshisft" prefix

Reply:
anyway, I moved them to the bottom of the import statement :)